include/drivers/sensor/ccs811.h

```diff
@@ -20,6 +20,7 @@ extern "C" {
 #endif
 
 #include <device.h>
+#include <sensor.h>
```

Comment:
`drivers/sensor.h`

Reply:
Huh.  It works as is.  It probably shouldn't, but that's a build system problem: `CMakeLists.txt` is `include/drivers` in the include path.